src/server/app.js

```diff
@@ -32,7 +32,6 @@ if (app.get('env') === 'development') {
 // TODO: WE might want to move the middleware to each of the individual routes
 // so we don't have weird conflicts in the future.
 app.use('/elasticsearch', proxy);
-app.use('/enforcer', require('./lib/enforce'));
```

Comment:
Uh? Why has this been removed? This is integral to the operation of the server. One might even say it's the most important part of the server. @w33ble back me up on this